app/controllers/publishers/vacancies/job_applications_controller.rb

```diff
@@ -30,6 +31,8 @@ def update_status
     redirect_to organisation_job_job_applications_path(vacancy.id), success: t(".#{status}", name: job_application.name)
   end
 
+  def withdrawn; end
```

Comment:
I think you can leave this out! Rails will create it.

Reply:
No need to shout!